Tabnine/src/main/java/com/tabnine/chat/commandHandlers/InitHandler.kt

```diff
@@ -0,0 +1,40 @@
+import com.google.gson.Gson
+import com.google.gson.JsonElement
+import com.intellij.openapi.editor.colors.EditorColorsManager
+import com.intellij.openapi.editor.colors.EditorFontType
+import com.intellij.openapi.project.Project
+import com.tabnine.chat.commandHandlers.ChatMessageHandler
+import java.awt.Color
+import javax.swing.UIManager
+
+data class InitPayload(
+    private val ide: String,
+    private val isDarkTheme: Boolean,
+    private val colors: MutableMap<String, String>,
+    private val fontSize: Int,
+)
+
+class InitHandler(gson: Gson) : ChatMessageHandler<Unit, InitPayload>(gson) {
+    override fun handle(payload: Unit?, project: Project): InitPayload {
+        val colorPalette = readColorPalette()
+        val isDarkTheme = EditorColorsManager.getInstance().isDarkEditor
+        val font = EditorColorsManager.getInstance().globalScheme.getFont(EditorFontType.PLAIN).size
```

Comment:
@yairco1990 note that this is representing the `Editor -> Font -> Size` setting property, so it should be fine, but it doesn't adapt to ctrl+mouse wheel font changes